docker-compose.yml

```diff
@@ -8,6 +8,7 @@ services:
         BASE_DIR: /app/
     volumes:
       - ./test/mysql-config:/etc/mysql/conf.d
+      - ./raisinbread/test/data-raisinbread/:/data-raisinbread
```

Comment:
this line is not necessary.

Reply:
I did try to remove it (and the others you marked as not necessary), but then it didn't work anymore. Unfortunately, I am unable to test this change right now. As I think you may have noticed, there is a problem with `composer` packages that makes Travis unable to run checks (ends up being errored). This is also causing errors when I try to run tests today...

I tried to add the ` composer/package-versions-deprecated` package as suggested and it did solve the composer packages problem, but it shows a lot of new valid formatting problems that were not detected before.

Reply:
I will fix the composer issue. Then you can test it again.

Reply:
I was able to run the tests and it looks like you were right, it works without the 3 lines you said were not necessary. I removed them.